plugin/kubernetes/container.go

```diff
@@ -0,0 +1,150 @@
+package kubernetes
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/pkg/errors"
+	"github.com/puppetlabs/wash/activity"
+	"github.com/puppetlabs/wash/plugin"
+	corev1 "k8s.io/api/core/v1"
+	k8s "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/remotecommand"
+	k8exec "k8s.io/client-go/util/exec"
+)
+
+type container struct {
+	plugin.EntryBase
+	client *k8s.Clientset
+	config *rest.Config
+	ns	 string
+	pod *corev1.Pod
+}
+
+func newContainer(ctx context.Context, client *k8s.Clientset, config *rest.Config, ns string, c *corev1.Container, p *corev1.Pod) (*container, error) {
+	cntnr := &container{
+		EntryBase: plugin.NewEntry(c.Name),
+	}
+	cntnr.client = client
+	cntnr.config = config
+	cntnr.ns = ns
+	cntnr.pod = p
+
+	// TODO: Get container startup time from ContainerStatus from the parent Pod
```

Comment:
How easy is it to add this?

Reply:
The metadata is located somewhere in the depths of the pod. I think `Pod` has a `PodStatus` which has a set of `ContainerStatus` which include these times. If the metadata isn't necessary we can defer it for later. I just noticed that some other objects had creation time.

Reply:
Fixed.